recipes/openssl/1.x.x/test_package/CMakeLists.txt

```diff
@@ -15,20 +15,9 @@ endif()
 if(USE_FIND_PACKAGE)
     set(OpenSSL_DEBUG 1)
     find_package(OpenSSL REQUIRED)
-    message("LINK WITH ${OPENSSL_LIBRARIES}")
+    message("LINK WITH ${OpenSSL_LIBRARIES}")
```

Comment:
https://cmake.org/cmake/help/v3.6/module/FindOpenSSL.html 

Variables in make should always be caps

Reply:
Wasn't the original cmake variable correct? All caps, like it is written in the documentation.

Reply:
Frankly I don't remember, but I don't think it was correct because it is generated with the component name, right ? And I didn't changed it in this PR. I didn't break anything here, I just fixed the message here.

Reply:
On one side, you're correct about fixing the message.
On the other side, it obscures the problem.

I think there should be a FIXME added mentioning that the cmake variables have incorrect casing.
That way, we remember there is a problem.

Reply:
Correct, and OpenSSL is not alone. Protobuf also diverges it. Unfortunately, we don't have such feature in conan to fix (yet, but requires a feature request first).